### PR TITLE
Notebooks: Support all positions

### LINKED ISSUE #828: Support all notebook positions
Screenshot from 2020-09-15 12 05 53@2x

### PR DESCRIPTION
**Pull request by danrabbit**
Fixes #828 

Screenshot from 2020-09-15 15 34 02@2x

### CODE REVIEW

File: src/widgets/_notebooks.scss [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
+        &.left {
+            background-image:
+                linear-gradient(
+                    to right,
+                    rgba(black, 0),
+                    rgba(black, 0) calc(100% - #{rem(2px)}),
+                    rgba(black, 0.05) calc(100% - #{rem(1px)}),
+                    rgba(black, 0.15)
+                );
+        }
```
Comment (cassidyjames) [inline src/widgets/_notebooks.scss:47]: Can we DRY this out a bit?

```suggestion
        $gradient: 
            rgba(black, 0),
            rgba(black, 0) calc(100% - #{rem(2px)}),
            rgba(black, 0.05) calc(100% - #{rem(1px)}),
            rgba(black, 0.15);

        &.top {
            background-image: linear-gradient(to bottom, $gradient);
        }

        &.bottom {
            background-image: linear-gradient(to top, $gradient);
        }

        &.right {
            background-image: linear-gradient(to left, $gradient);
        }

        &.left {
            background-image: linear-gradient(to right, $gradient);
        }
```

**Review by cassidyjames — CHANGES_REQUESTED** [review]
DRY

**Comment by danrabbit** [comment]
Ah yes. Im still thinking in CSS sometimes

**Review by cassidyjames — APPROVED** [review]
Nice

**Pull request merged**